jobserver/migrations/0019_add_backend_missing_threshold.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.0.4 on 2022-06-23 10:28
+
+import datetime
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("jobserver", "0018_move_project_numbers_to_number_field"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="backend",
+            name="missing_threshold",
```

Comment:
nit: I think "alert_timeout" might be a better name? I think it better describes what it does, "missing" is a bit vague?